src/main/java/com/mraof/minestuck/client/ClientProxy.java

```diff
@@ -67,6 +70,7 @@ public static void init()
 		RenderingRegistry.registerEntityRenderingHandler(VitalityGelEntity.class, VitalityGelRenderer::new);
 		RenderingRegistry.registerEntityRenderingHandler(DecoyEntity.class, DecoyRenderer::new);
 		RenderingRegistry.registerEntityRenderingHandler(MetalBoatEntity.class, MetalBoatRenderer::new);
+		RenderingRegistry.registerEntityRenderingHandler(BarbasolBombEntity.class, manager -> new SpriteRenderer<>(manager, Minecraft.getInstance().getItemRenderer()));
```

Comment:
I do not know how the SpriteRenderer works, but I take it that you've tested it and made sure that this works?

Reply:
yes, it does